app/controllers/catalog_controller.rb

```diff
@@ -61,14 +61,40 @@ def show
     super
   end
 
+  def report_issue_action(documents)
+    mail = RecordMailer.email_report_issue(documents, {
```

Comment:
We're actually not using the `documents` variable here at all. When we do the `config.add_tools_partial` thing, Blacklight is generating a method called `report_issue`. That method that sets the `@documents` variable that you're using in your form. This is 👍 . But, when we make the callback to `report_issue_action`, because Blacklight has _generated_ the `report_issue` method, it's passing `@documents` into our callback. We can't change that...unless we write our own `report_issue` method, but nah... we'll just discard the documents parameter here, and only pass what the email needs:

``` ruby
def report_issue_action(_documents)                                                                                      
    mail = RecordMailer.email_report_issue(                                                                                
      title: params[:title],                                                                                               
      record_number: params[:record_number],                                                                               
      comment: params[:comment],                                                                                           
      email: params[:email],                                                                                               
      url_options: url_options)
```

This will change the method signature of `email_report_issue` slight because we're using keyword arguments:

``` ruby
def email_report_issue(title:, record_number:, comment:, email:, url_gen_params:)
```